torch/csrc/jit/codegen/cuda/kernel_cache.cpp

```diff
@@ -47,9 +47,13 @@ void debugPrint(const TensorTypePtr& type) {
     for (int i = 0; i < rank; i++) {
       const auto& shape_symbol = sizes.value()[i];
       if (shape_symbol.is_static()) {
-        printf("%ld, ", shape_symbol.static_size());
+        // int64_t format: convert upward to long long int, so formatter to lld
+        printf("%lld, ", (long long int)shape_symbol.static_size());
       } else {
-        printf("s(%ld), ", *reinterpret_cast<const int64_t*>(&shape_symbol));
+        // int64_t format: convert upward to long long int, so formatter to lld
+        printf(
+            "s(%lld), ",
+            (long long int)*reinterpret_cast<const int64_t*>(&shape_symbol));
```

Comment:
Why not cast to `long long int` int the first place?
```suggestion
            *reinterpret_cast<const long long int*>(&shape_symbol));
```